client/gutenberg/extensions/presets/jetpack/package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "@automattic/jetpack-blocks",
+	"version": "0.1.0-alpha.1",
+	"description": "Gutenberg Blocks for the Jetpack WordPress Plugin",
+	"main": "editor.js",
+	"publishConfig": {
+		"access": "public"
+	},
+	"scripts": {
+		"build": "../../../../../bin/sdk-cli.js gutenberg .",
+		"prepublishOnly": "npm run build",
+		"test": "echo \"Error: no test specified\" && exit 1"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git"
+	},
+	"keywords": [
+		"jetpack",
+		"gutenberg",
+		"wordpress",
+		"blocks"
+	],
+	"author": "Automattic",
+	"license": "GPL-2.0",
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"homepage": "https://github.com/Automattic/wp-calypso#readme"
```

Comment:
Good point. I'm adding the workflow doc to the Jetpack repo (https://github.com/Automattic/jetpack/pull/10276), since a lot of steps are Jetpack specific.

For this package's README, I'd like to something that is
- not entirely SDK specific -- the SDK isn't the code that this package is going to contain. Instead, the README should contain information pertaining to Jetpack Gutenblock (e.g. a list of the blocks it contains)
- found inside this directory.

I think I'll add a README stub to this dir. As soon as https://github.com/Automattic/jetpack/pull/10276 is merged, we can add a link to that doc to this README.